### DISCUSSION TITLE
docs: hint to use Linux install instructions for WSL over Windows/general stance on WSL support

### DISCUSSION THREAD

**Discussion by Encephala** [zed-industries/zed#12931] [Feature Requests]
I wanted to test Zed through WSL and blindly started following the Windows instructions (fair enough, my bad).
I ended up manually installing various system dependencies and the process felt quite bad, until I found an issue on this repo talking about the Linux install script (`script/linux`) and realised I should've been using that.

I figure this is worth a small note in the docs, for example [like this](https://github.com/Encephala/zed/commit/e77afeda243afdc609dcc2b65137eaee7c2e14b6).
However, I'm not sure what your stance is on WSL support - I got various runtime errors that weren't easily fixed by installing a package, but otherwise the editor seemed to work about as well as it did on native Windows, so perhaps a note on your stance somewhere in the docs is worthwhile. A quick grep through the docs doesn't return anything, nor is there any mention of it in the platform support issue, bar [this old user comment](https://github.com/zed-industries/zed/issues/5395#issuecomment-1908667183).

I'm happy to open a PR, or feel free to add a comment or something yourselves.

**Comment by airtonix** [comment]
Unless you're a masochist, most people should now just install the windows version and connect to a wsl instance with `ctrl + shift + alt + o`

**Reply by vsrnth** [reply]
I had no clue we could do this. Thanks a bunch for posting this.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links

**Reply by airtonix** [reply]
sounds like a bad move. discord is SUPER horrible at storing knowledge.
